manager/src/main/resources/application.properties

```diff
@@ -102,7 +102,7 @@ quarkus.kafka.health.enabled=false
 mp.messaging.connector.smallrye-kafka.health-enabled=false
 
 # Managed Connectors
-managed-connectors.cluster.id=${MC_CLUSTER_ID:fakeClusterId}
+managed-connectors.namespace.id=${MC_CLUSTER_ID:fakeNamespaceId}
```

Comment:
Does it mean that namespace id == cluster id?So it is a same value, just renamed property?

Reply:
It was wrong, I fixed